### DISCUSSION TITLE
Information

### DISCUSSION THREAD

**Discussion by Cbarneron1** [CollaboraOnline/online#4205] [Q&A] [answered]
Hello ,
We Begin to use Collabora for Moodle site  and  we need information  on the limits  to the version CODE of the plugin.
 If he has a limit to document use or to document open.
 If he have a limit to user connect to the same document.

Thanks  for the answer

**Comment by pedropintosilva** [comment, marked as answer]
@Cbarneron1 we used GH discussions mainly for some very specific announcements. Could you please re-post this in the forum https://forum.collaboraonline.com/ in there we have much more information and many questions answered : )

**Reply by Cbarneron1** [reply]
ok no probleme , thanks for the answer
